src/main/java/com/elastic/support/util/RestExec.java

```diff
@@ -91,10 +91,10 @@ protected HttpResponse exec(String url) {
          }
       } catch (HttpHostConnectException e) {
          logger.log(SystemProperties.DIAG, "Host connection error.", e);
-         throw new RuntimeException("Error connecting to host " + url, e);
+         throw new RuntimeException("Host connection error: " + e.getMessage() );
```

Comment:
This drops the URL (unless it's always printed by the exception?) and also eliminates any caused by exceptions printable in the logs. I assume that's intentional?

Reply:
See above. The URL is actually in the message for this type of error. I've already logged the stack trace in the internal log.